src/main/java/com/ericsson/ei/controller/AuthControllerImpl.java

```diff
@@ -66,7 +68,7 @@
         } catch (Exception e) {
             String errorMessage = "Failed to log in user. Error message:\n" + e.getMessage();
             LOGGER.error(errorMessage, e);
-            return new ResponseEntity<>(errorMessage, HttpStatus.INTERNAL_SERVER_ERROR);
+            return new ResponseEntity<>(ResponseMessage.createJsonMessage(errorMessage), HttpStatus.INTERNAL_SERVER_ERROR);
```

Comment:
Same here, if this is an internal server error, then we should do something similar to my previous comment. Why shouldn't give internal information back to the user. Same with the rest of the similar code :)

Reply:
Done.